src/injectHighlightWrappers.ts

```diff
@@ -18,6 +18,12 @@ const IGNORE_TAGS = [
   'SCRIPT', 'STYLE', 'SELECT', 'OPTION', 'BUTTON', 'OBJECT', 'APPLET',
   'VIDEO', 'AUDIO', 'CANVAS', 'EMBED', 'PARAM', 'METER', 'PROGRESS',
 ];
+/**
+ * Highlights can be created around these block and text elements.
+ */
+const BLOCK_ELEMENTS = ['img', 'video', 'iframe'];
+const TEXT_ELEMENTS = ['.MathJax'];
+const ALLOWED_ELEMENT = BLOCK_ELEMENTS.concat(TEXT_ELEMENTS);
```

Comment:
not sure if `video` should be both in ignored and allowed tags.

Reply:
Nicely spotted :D